uber/config.py

```diff
@@ -394,6 +419,18 @@ def _is_intstr(s):
 c.HOURS = ['{:02}'.format(i) for i in range(24)]
 c.MINUTES = ['{:02}'.format(i) for i in range(60)]
 
+c.DAYS_OF_WEEK = {'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'}
+if c.ONE_DAYS_ENABLED and c.PRESELL_ONE_DAYS:
+    _day = c.EPOCH.date()
+    while _day <= c.ESCHATON.date():
+        _name = _day.strftime('%A')
+        _val = c.create_enum_val(_name)
+        c.BADGES[_val] = _name
+        c.BADGE_OPTS.append((_val, _name))
+        c.BADGE_VARS.append(_name.upper())
+        c.BADGE_RANGES[_val] = c.BADGE_RANGES[c.ONE_DAY_BADGE]
+        _day += timedelta(days=1)
```

Comment:
What I'm about to explain is documented in the comments in ``configspec.ini`` but I'll repeat it here because this is a slight change from what I proposed in an earlier comment in the discussion on this ticket.

If you set ``presell_one_days`` to true, this code automatically inserts a badge type for each day of the event.  In out case that means we have ``c.THURSDAY``, ``c.FRIDAY``, ``c.SATURDAY``, and ``c.SUNDAY`` badge types automatically configured (based on the ``c.EPOCH`` and ``c.ESCHATON`` settings).  We do NOT have to add new options to the ``[[badge]]`` config subsection of the ``[enums]`` config section.